code/modules/projectiles/projectile/bullets/sniper.dm

```diff
@@ -1,13 +1,13 @@
 // .50 (Sniper)
 
 /obj/item/projectile/bullet/p50
-	name =".50 bullet"
+	name =".50 BMG bullet"
 	pixels_per_second = TILES_TO_PIXELS(25)
-	damage = 70
+	damage = 92                    //Skyrat Edit
 	knockdown = 100
 	dismemberment = 50
```

Comment:
Either the bullet goes clean through (AP), or it dumps all of its energy upon impact (blows off limbs). It probably shouldn't do both, right?

Mechanically, I don't think the gun needs a 50% chance to dismember someone on top of nearly 1-shotting them.

Reply:
The desired effect for it was to convey the feeling that no reasonable mount of armor will stop a .50 cal from damaging you one way or another, ideally, it could've kept the previous AP value with a added variables to deal considerable organ damage to every organ in the torso, but I have no clue if that will even work.

Point in case, the sniper rifle is designed to cuck someone in one hit to reward its general unwieldiness, extremely obvious sprite, size and speed. Same goes for the Anti-Tank pistol. If it wasn't for the sake of balance, I'd bump it up to 120, but the game needs to be fun.